coding_style/web.md

```diff
@@ -20,3 +20,8 @@ Use [Prettier](https://github.com/prettier/prettier)
 
 * `.js` is only standard JS.
 * `.jsx` is files contain JSX.
+
+## Role of package.json in each component directory
+
+package.json file has a role in navigating to main component file.
+Although `index.js` has same role, it follow convention of react-starter-kit.
```

Comment:
[Q] We can use `We follow the convention`?

Reply:
sounds good!
thanks 🎉